docs/diagrams/InnerWorkings.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Package Client {
+Class Client
+Class Address
+Class Email
+Class Name
+Class Phone
+Class UniqueClientList
+Class ContractExpiryDate
+Class Timezone
+}
+
+Package Country {
+Class Country
+Class CountryManager
+}
+
+Package Note {
+Class Note
+Class TagNoteMap
+}
+
+Package Tag {
+Class Tag
+Class UniqueTagSet
+}
+
+AddressBook *-right-> "1" UniqueClientList
+AddressBook *-right-> "1" UniqueTagSet
+AddressBook *-right-> "1" CountryManager
+
+UniqueClientList o--> "*" Client
+
+Client *--> Name
+Client *--> Phone
+Client *--> Email
+Client *--> Address
+Client *--> ContractExpiryDate
+Client *-up-> "*" Tag
+Client -up-> Country
+Client --> Timezone
+Client -->"*" Note
+
+CountryManager o-left->"*" Country : has >
+UniqueTagSet o-right->"*" Tag
+TagNoteMap o-right->"*" Note
+
+TagNoteMap o-up-> Tag
+CountryManager --> "*" Note : creates mapping >
+
+Client -[hidden]right- AddressBook
+@enduml
+
+
```

Comment:
extra newline

Reply:
> I think for ModelClassDiagram, we might needa add arrows from addressbook to countrynotesmanager

It will make the diagram look pertty ugly

Reply:
nvm i found a good orientation